src/api_crypto.rs

```diff
@@ -120,10 +120,14 @@ impl VerifyKey {
         signature: &[u8],
         message: &[u8],
     ) -> PyResult<&'py PyBytes> {
-        match self.0.verify_with_signature(signature, message) {
-            Ok(v) => Ok(PyBytes::new(py, &v)),
-            Err(_) => Err(CryptoError::new_err("Signature was forged or corrupt")),
-        }
+        self.0
+            .verify_with_signature(
+                <[u8; 64]>::try_from(signature)
```

Comment:
64 ? why is this 64 ? there should be at least a comment to explain this magic number !

And it would be much better to use a constant, e.g. `libparsec::crypto::VerifyKey::SIGNATURE_BYTE_SIZE`